api/SVGGraphicsElement.json

```diff
@@ -200,10 +200,10 @@
         "__compat": {
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "43"
```

Comment:
Attributes on prototypes strike again. I've finally filed https://github.com/mdn/browser-compat-data/issues/7843, should have done that a long time ago, since I keep forgetting what the magic version number is where this happens.